phpBB/styles/prosilver/theme/forms.css

```diff
@@ -210,19 +204,20 @@ fieldset.display-options a {
 
 /* Display actions for ucp and mcp pages */
 fieldset.display-actions {
-	line-height: 2em;
+	line-height: 22px;
 	text-align: right;
 	white-space: nowrap;
-	padding-right: 1em;
+	padding-right: 11px;
 }
 
 fieldset.display-actions label {
 	white-space: nowrap;
 	padding-right: 2px;
 }
 
+/* Not used anywhere */
```

Comment:
possibly, but its always been display-options for that https://github.com/phpbb/phpbb/blame/3.0.x/phpBB/styles/prosilver/template/viewtopic_body.html#L227